photon-core/src/main/java/org/photonvision/vision/apriltag/AprilTagDetectorParams.java

```diff
@@ -30,7 +30,7 @@
 
 public class AprilTagDetectorParams {
     public static AprilTagDetectorParams DEFAULT_36H11 =
```

Comment:
Should this variable be named `DEFAULT_36H11` given that the set tag family is `kTag16h5`?